app/scripts/metamask-controller.js

```diff
@@ -988,25 +1137,47 @@ module.exports = class MetamaskController extends EventEmitter {
     }
   }
 
+  /*
+   * A method for forwarding the user to the easiest way to obtain ether,
+   * or the network "gas" currency, for the current selected network.
+   *
+   * @param {String} address - The address to fund.
+   * @param {String} amount - The amount of ether desired, as a base 10 string.
+   */
   buyEth (address, amount) {
     if (!amount) amount = '5'
     const network = this.networkController.getNetworkState()
     const url = getBuyEthUrl({ network, address, amount })
     if (url) this.platform.openWindow({ url })
   }
 
+  /*
+   * A method for triggering a shapeshift currency transfer.
+   * @param {String} depositAddress - The address to deposit to.
```

Comment:
`{String}` should be lowercase. JSDoc has a "strict" mode that can error on these types of issues, but I don't think we're using it.